hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -300,7 +300,7 @@
     DFS_NAMENODE_KERBEROS_PRINCIPAL_KEY,
     DFS_NAMENODE_KERBEROS_INTERNAL_SPNEGO_PRINCIPAL_KEY,
     DFS_HA_FENCE_METHODS_KEY,
-    DFS_HA_ZKFC_PORT_KEY,
```

Comment:
What is the issue with having a comma at the end of an array?

Reply:
@goiri The issue HDFS-9929 forget to delete the comma.

Reply:
Does anything break because of this comma?
I personally don't see a point on removing this.
If we add a new key, it will reduce churn.

Reply:
@goiri It is not break anything, I think the issue opener means the redundant comma is not formal java grammar.